book/src/install.md

```diff
@@ -143,33 +141,35 @@ pacman -S mingw-w64-ucrt-x86_64-helix
 
 ## Building from source
 
-Clone the repository:
+Requirements:
+
+- The [Rust toolchain](https://www.rust-lang.org/tools/install)
+- A c++14 compatible compiler to build the tree-sitter grammars, for example `gcc-c++`
```

Comment:
Is `gcc-c++` a debian package? I don't think the reference makes sense for other OSs, for example Arch or macos through brew. We might be able to say something generic like "GCC or Clang" if `gcc-c++` is just g++

Reply:
Yes much better to make it generic. `gcc-c++` was just the package I used:

```
Installed Packages
Name         : gcc-c++
Version      : 11.3.1
Release      : 2.1.el9.alma
Architecture : x86_64
Size         : 32 M
Source       : gcc-11.3.1-2.1.el9.alma.src.rpm
Repository   : @System
From repo    : appstream
Summary      : C++ support for GCC
URL          : http://gcc.gnu.org
License      : GPLv3+ and GPLv3+ with exceptions and GPLv2+ with exceptions and LGPLv2+ and BSD
Description  : This package adds C++ support to the GNU Compiler Collection.
             : It includes support for most of the current C++ specification,
             : including templates and exception handling.
```